### PR TITLE
feat: peregrine spec

### PR DESCRIPTION
**Pull request by weichweich**
## related to KILTProtocol/ticket#1227

* adds a feature that reduces the duration of all governance related eras to a few minutes
* change specs with JQ

## Checklist:

- [x] I have verified that the code works
  - [ ] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] This PR does not introduce new custom types
  - [ ] If not, I have opened a companion PR with the type changes in the [KILT types-definitions repository](https://github.com/KILTprotocol/type-definitions/pulls)
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: nodes/parachain/src/chain_spec.rs [hunk 1]
```diff
@@ -146,6 +146,36 @@ pub fn staging_test_net(id: ParaId) -> Result<ChainSpec, String> {
 	))
 }
 
+pub fn peregrine_test_net(id: ParaId) -> Result<ChainSpec, String> {
+	let properties = get_properties("PKILT", 15, 38);
```
Comment (Diiaablo95) [inline nodes/parachain/src/chain_spec.rs:150]: What are these 15 and 38?

Comment (Diiaablo95) [reply nodes/parachain/src/chain_spec.rs:150]: And also PKILT?

Comment (weichweich) [reply nodes/parachain/src/chain_spec.rs:150]: PKILT = peregrine-kilt

15 = 10^15 units are one PKILT

38 = The SS58 identifier

Yeah it's not that readable. But i would hope that's something for a different PR? 😅  We do it for all specs like that.

File: scripts/peregrine-relay.jq [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.bootNodes = []
```
Comment (Diiaablo95) [inline scripts/peregrine-relay.jq:1]: Love this approach! 😎

File: scripts/spec.sh [hunk 3]
```diff
@@ -1,17 +1,33 @@
...
 
+ROCOCO_PLAIN=/tmp/parachain/rococo.plain.json
+ROCOCO=/tmp/parachain/rococo.json
+
+# ##############################################################################
+# #                                                                            #
+# #                                  PEREGRINE                                 #
+# #                                                                            #
+# ##############################################################################
+docker run parity/rococo:rococo-v1-0.8.30-943038a8-f14fa75f build-spec --chain rococo-local --disable-default-bootnode >$ROCOCO_PLAIN
```
Comment (Diiaablo95) [inline scripts/spec.sh:12]: I would leave a space before and after the redirect symbol ">".

File: scripts/spec.sh [hunk 4]
```diff
@@ -1,17 +1,33 @@
...
+cargo run --release -p kilt-parachain --features fast-gov -- build-spec --chain peregrine --disable-default-bootnode >peregrine-kilt.plain.spec
```
Comment (Diiaablo95) [inline scripts/spec.sh:13]: Ditto

**Review by Diiaablo95 — APPROVED** [review]
Few minor comments, but I guess LVVGTM 😁

File: scripts/spec.sh (hunk 3 above)
Comment (weichweich) [reply scripts/spec.sh]: I would too, but i have a weird bash formatter that removes it. 🤔 i added it. lets hope it survives the formatter for a while 😅

**Pull request merged**